cves/CVE-2012-3442.yml

```diff
@@ -109,10 +119,13 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    Author does not disclose how they discovered the vulnerability.
```

Comment:
The intent of the code/code_answer is not how the vulnerability was discovered, but whether the component was being unit tested.